app/proto/stripe.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
```

Comment:
Move file to backend-specific place

Reply:
This is not backend specific: it needs to be publicly available. It won't be called by users, but it'll be called by the stripe servers, and the proxy needs to do the translation. We could potentially create a new set of proto files that are public-facing but not part of the API used by users, but I think that's more hassle?

Reply:
Aha, it is not a protobuf-based protocol at all, and documented at https://stripe.com/docs/api/events/types . How can we do this without a .proto file?